### PR TITLE
Updated kfserving base image

### LINKED ISSUE #2236: Update to use KFserving 0.4.0 artifacts
KFServing released 0.4.0 https://github.com/kubeflow/kfserving/releases, we should update to use their latest storage initialiser images to fix existing bugs, as well as update other refernces that point to older kfserving versions.

### PR DESCRIPTION
**Pull request by axsaucedo**
Fixes #2236

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml [hunk 1]
```diff
@@ -67,6 +67,8 @@ spec:
           value: ''
         - name: RELATED_IMAGE_EXPLAINER
           value: ''
+        - name: RELATED_IMAGE_MOCK_CLASSIFIER
```
Comment (axsaucedo) [inline helm-charts/seldon-core-operator/templates/deployment_seldon-controller-manager.yaml:70]: It seems this change wasn't reflected from another PR

**Comment by seldondev** [comment]
Mon Aug 10 12:43:49 UTC 2020
The logs for [integration] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=3

**Comment by seldondev** [comment]
Mon Aug 10 12:43:53 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=2

**Comment by seldondev** [comment]
Mon Aug 10 12:44:05 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=1

**Comment by seldondev** [comment]
@axsaucedo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 6a9f52a49b7a0b0d9f7c6a33b0e93b05429bc5d5 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Mon Aug 10 12:52:58 UTC 2020
The logs for [integration] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=6

**Comment by seldondev** [comment]
Mon Aug 10 14:26:43 UTC 2020
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=7

**Comment by seldondev** [comment]
@axsaucedo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 6a9f52a49b7a0b0d9f7c6a33b0e93b05429bc5d5 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by axsaucedo** [comment]
It seems failed ones have been flaky ones (rolling update) - retesting
/test integration

**Comment by seldondev** [comment]
Mon Aug 10 15:40:04 UTC 2020
The logs for [integration] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2261/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2261 --build=8

**Comment by seldondev** [comment]
@axsaucedo: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 6a9f52a49b7a0b0d9f7c6a33b0e93b05429bc5d5 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by axsaucedo** [comment]
It seems all tests pass except test `test_label_update`, I will run locally to assess

**Pull request merged**